### ISSUE TITLE
Narrator announce ShimmeredDetailsList header twice

### ISSUE THREAD

**Issue by t-oferl** [microsoft/fluentui#18467]
### Environment Information

- **Package version(s)**: 7.165.2
- **Browser and OS versions**: Edge 91.0.864.41

### Describe the issue:

Narrator announce ShimmeredDetailsList header twice

#### Actual behavior:

when navigating to a ShimmeredDetailsListcolumn header, the Narrator announce it twice, it calls for the value one, and then the column header once

#### Expected behavior:

Narrator should only call for the column header once

### If applicable, please provide a codepen repro:

https://codepen.io/t-oferl/pen/Popaama
(Took the example from here: https://docs.microsoft.com/en-us/javascript/api/examples/detailslist/shimmer?view=office-ui-fabric-react-latest)

### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No)

Requested priority: (Blocking, High, Normal, Low)

Products/sites affected: (if applicable)

**Comment by gouttierre** [comment]
@t-oferl - Thanks for filing this issue. I tested your stated problem with our [latest fluent version 8](https://developer.microsoft.com/en-us/fluentui?fabricVer=8#/controls/web/shimmer) and was unable to repro your problem. Could you confirm using our latest version? Thanks

cc @smhigley, @miroslavstastny
